bitnami/redis/templates/metrics-sentinel-prometheus.yaml

```
@@ -0,0 +1,33 @@
+{{- if and .Values.sentinel.enabled .Values.sentinel.metrics.enabled .Values.sentinel.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "redis.fullname" . }}-sentinel-metrics
+  {{- if .Values.sentinel.metrics.serviceMonitor.namespace }}
+  namespace: {{ .Values.sentinel.metrics.serviceMonitor.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace | quote }}
+  {{- end }}
+  labels:
+    app: {{ template "redis.name" . }}
+    chart: {{ template "redis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    {{- range $key, $value := .Values.sentinel.metrics.serviceMonitor.selector }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  endpoints:
+  - port: sentinelmetrics
+    {{- if .Values.sentinel.metrics.serviceMonitor.interval }}
+    interval: {{ .Values.sentinel.metrics.serviceMonitor.interval }}
+    {{- end }}
```

Indentation (for consistency with other charts)
```suggestion
    - port: sentinelmetrics
      {{- if .Values.sentinel.metrics.serviceMonitor.interval }}
      interval: {{ .Values.sentinel.metrics.serviceMonitor.interval }}
      {{- end }}
```